Description: Add possibility of overriding of JVM_PATH into JWSGI buildscript
Author: Leonid Borisenko <leo.borisenko@gmail.com>
Bug: http://projects.unbit.it/uwsgi/ticket/11
Last-Update: 2011-06-10

Index: uwsgi/plugins/jwsgi/uwsgiplugin.py
===================================================================
--- uwsgi.orig/plugins/jwsgi/uwsgiplugin.py	2011-05-03 02:55:50.000000000 +0300
+++ uwsgi/plugins/jwsgi/uwsgiplugin.py	2011-05-03 02:56:54.000000000 +0300
@@ -10,6 +10,15 @@
 JVM_INCPATH = "/usr/lib/jvm/java-6-sun-1.6.0.15/include/ -I/usr/lib/jvm/java-6-sun-1.6.0.15/include/linux"
 JVM_LIBPATH = "/usr/lib/jvm/java-6-sun-1.6.0.15/jre/lib/i386/server/"
 
+try:
+    JVM_INCPATH = os.environ['UWSGICONFIG_JVM_INCPATH']
+except:
+    pass
+
+try:
+    JVM_LIBPATH = os.environ['UWSGICONFIG_JVM_LIBPATH']
+except:
+    pass
 
 CFLAGS = ['-I' + JVM_INCPATH]
 LDFLAGS = ['-L' + JVM_LIBPATH]
